yb-voyager/src/srcdb/common.go

```diff
@@ -82,3 +83,43 @@ func processImportDirectives(fileName string) error {
 	}
 	return nil
 }
+//Invoked at the end of Export Schema for Oracle for SYNONYM Object type to strip the source schema name from the sqlStatements
+func processStmtsToStripSourceSchema(fileName string, sourceSchema string) error {
```

Comment:
The `processStmtsTo` part is not required. Just `stripSourceSchemaNames` is enough.